### ISSUE TITLE
setup.mail.io generates non-working docker-compose.yml

### ISSUE THREAD

**Issue by wungad** [Mailu/Mailu#1818]
Visiting https://setup.mailu.io/ redirects to https://setup.mailu.io/1_7/ which generates docker-compose.yml with reference to mailu images with underscore in their tag. Such image does not exist on dockerhub thus producing the following error:

```
$ docker-compose up
...
Pulling front (mailu/nginx:1_7)...
ERROR: manifest for mailu/nginx:1_7 not found: manifest unknown: manifest unknown
```
Replacing the underscore with dot (1_7 to 1.7) solves this issue:

```
$ docker-compose up
...
Pulling front (mailu/nginx:1.7)...
Status: Downloaded newer image for mailu/nginx:1.7
```

**Comment by anver** [comment]
> Visiting https://setup.mailu.io/ redirects to https://setup.mailu.io/1_7/ which generates docker-compose.yml with reference to mailu images with underscore in their tag. Such image does not exist on dockerhub thus producing the following error:
> 
> ```
> $ docker-compose up
> ...
> Pulling front (mailu/nginx:1_7)...
> ERROR: manifest for mailu/nginx:1_7 not found: manifest unknown: manifest unknown
> ```
> 
> Replacing the underscore with dot (1_7 to 1.7) solves this issue:
> 
> ```
> $ docker-compose up
> ...
> Pulling front (mailu/nginx:1.7)...
> Status: Downloaded newer image for mailu/nginx:1.7
> ```

Yes same happened for me and the '.' fixes the issue, @wungad  thx

**Comment by Hosni** [comment]
> > Visiting https://setup.mailu.io/ redirects to https://setup.mailu.io/1_7/ which generates docker-compose.yml with reference to mailu images with underscore in their tag. Such image does not exist on dockerhub thus producing the following error:
> > ```
> > $ docker-compose up
> > ...
> > Pulling front (mailu/nginx:1_7)...
> > ERROR: manifest for mailu/nginx:1_7 not found: manifest unknown: manifest unknown
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Replacing the underscore with dot (1_7 to 1.7) solves this issue:
> > ```
> > $ docker-compose up
> > ...
> > Pulling front (mailu/nginx:1.7)...
> > Status: Downloaded newer image for mailu/nginx:1.7
> > ```
> 
> Yes same happened for me and the '.' fixes the issue, @wungad thx

Same Here!, this worked for me also.

**Comment by rafelck** [comment]
> Visiting https://setup.mailu.io/ redirects to https://setup.mailu.io/1_7/ which generates docker-compose.yml with reference to mailu images with underscore in their tag. Such image does not exist on dockerhub thus producing the following error:
> 
> ```
> $ docker-compose up
> ...
> Pulling front (mailu/nginx:1_7)...
> ERROR: manifest for mailu/nginx:1_7 not found: manifest unknown: manifest unknown
> ```
> 
> Replacing the underscore with dot (1_7 to 1.7) solves this issue:
> 
> ```
> $ docker-compose up
> ...
> Pulling front (mailu/nginx:1.7)...
> Status: Downloaded newer image for mailu/nginx:1.7
> ```

Thank's dude

**Comment by Xat59** [comment]
I can confirm this issue. I've first fixed it by replacing `1_7` with `1.7` in the downloaded `docker-compose.yml`.

Thanks
